### PR TITLE
LPD-3259 Problem with viewing the Review Order page

### PR DESCRIPTION
**Pull request by michelevigilante**
https://liferay.atlassian.net/browse/LPD-3259

### CODE REVIEW

**Comment by michelevigilante** [comment]
ci:test:playwright

**Comment by michelevigilante** [comment]
ci:test:relevant

File: modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts [hunk 1]
```diff
@@ -66,6 +71,21 @@ export class CommerceAdminChannelsPage {
 		await this.applicationsMenuPage.goToCommerceChannels();
 	}
 
+	async changeBuyerOrderApprovalWorkflow(
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts:74]: `changeCommerceChannelBuyerOrderApprovalWorkflow`

File: modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts [hunk 2]
```diff
@@ -17,6 +18,7 @@ export class CommerceAdminChannelsPage {
 		strictEqual?: boolean
 	) => Promise<{column: Locator; row: Locator}>;
 	readonly channelsTableRowLink: (channelName: string) => Promise<Locator>;
+	readonly commerceApprovalWorkflow: Locator;
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts:21]: `buyerOrderApprovalWorkflow`

File: modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts [hunk 3]
```diff
@@ -66,6 +71,21 @@ export class CommerceAdminChannelsPage {
 		await this.applicationsMenuPage.goToCommerceChannels();
 	}
 
+	async changeBuyerOrderApprovalWorkflow(
+		channelName: string,
+		approvalWorkflow: string
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminChannelsPage.ts:76]: `buyerOrderApprovalWorkflow`

Also, pay attention to the alphabetical order.

File: modules/test/playwright/tests/commerce/commerce-cart-content-web/commerceCartSummary.spec.ts [hunk 4]
```diff
@@ -33,12 +35,12 @@ test('LPD-3360 Checkout with single approval', async ({
 	apiHelpers.data.push({id: site.id, type: 'site'});
 
 	const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
-		name: 'Cart Single Approval Channel',
+		name: `${site.name} Channel`,
```
Comment (alessiorendina) [inline modules/test/playwright/tests/commerce/commerce-cart-content-web/commerceCartSummary.spec.ts:38]: Let's make also site name random

File: modules/test/playwright/tests/commerce/commerce-order-content-web/pendingOrder.spec.ts [hunk 5]
```diff
@@ -746,3 +746,102 @@ test('LPD-26906 As a buyer, I can edit product options from the pending orders p
...
+		},
+		channels.items[0].id
+	);
+
+	await page.goto(`/web/${site.name}`);
+
+	await commerceMiniCartPage.miniCartButton.click();
+	await commerceMiniCartPage.reviewOrderButton.click();
+	await expect(pendingOrdersPage.orderItemsTable).toBeVisible();
+});
```
Comment (alessiorendina) [inline modules/test/playwright/tests/commerce/commerce-order-content-web/pendingOrder.spec.ts:847]: Missing the fix Issue on channel details.

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@michelevigilante Please check my comments, thank you!

**Comment by michelevigilante** [comment]
ci:test:playwright

File: modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts [hunk 6]
```diff
@@ -4,16 +4,27 @@
...
 	readonly closeSidePanelFrame: (
 		isNestedFrame: boolean,
 		tableName: string
 	) => Promise<Locator>;
+	readonly commerceChannelHealthChecksTable: Locator;
+	readonly commerceChannelHealthChecksTableRowAction: (
+		pageName: string,
+		action: string
+	) => Promise<Locator>;
+	readonly commerceChannelHealthChecksTableRow: (
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts:23]: Sort

File: modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts [hunk 7]
```diff
@@ -64,6 +75,41 @@ export class CommerceAdminChannelDetailsPage {
...
+			);
+
+			if (pagesTableRow && pagesTableRow.column) {
+				return pagesTableRow.row.getByRole('link', {
+					name: action,
+				});
+			}
+
+			throw new Error(
+				`Cannot locate page row with name ${pageName}`
```
Comment (alessiorendina) [inline modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts:110]: page row -> table row

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@michelevigilante Overall it looks good to me, just 2 small change requests. Thank you!

**Review by alessiorendina — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**